ui/js/auth/AuthService.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
```

Comment:
We're soon going to be switching JS from 4 space to 2 space indentation. We've already imported some 2 space code when we merged in Cameron's test-view work from another repo, and at that point turned off the eslint indent rule entirely to avoid churn. 

As such, if you'd like to use 2 space in this file and any other that's new in this PR, it will reduce the impact on future git blame when we mass convert. However if it's too much of a pain it's ok to leave as-is too.